src/comparator/wrappers.py

```diff
@@ -219,19 +219,20 @@ def required(self):
 
     @property
     def proto_type(self):
-        """Return the proto type constant e.g. `TYPE_ENUM`"""
-        # TODO(xiaozhenliu): convert proto type `TYPE_STRING` to `string`.
+        """Return the proto type constant e.g. `enum`"""
         return WithLocation(
-            FieldDescriptorProto().Type.Name(self.field_pb.type),
+            cast(str, FieldDescriptorProto().Type.Name(self.field_pb.type))[
+                len("TYPE_") :
+            ].lower(),
```

Comment:
What is the purpose of the `cast`? What does `cast` do in a dynamically typed language? I've never seen it before.

Reply:
I referenced this from python micro [here](https://github.com/googleapis/gapic-generator-python/blob/master/gapic/schema/wrappers.py#L161) and thought it might be cases that the type of `descriptor_pb2.FieldDescriptorProto.Type.Name()` is not str. But based on the current tests, seems it is ok to just remove the `cast`. Thanks

Reply:
Done.